purchase_batch_invoicing/tests/test_invoicing.py

```diff
@@ -3,35 +3,32 @@
 from unittest import mock
 
 from odoo.exceptions import UserError
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import SavepointCase, tagged
 
 mock_ns = "odoo.addons.purchase_batch_invoicing.wizards" ".purchase_batch_invoicing"
 
 
+@tagged("post_install", "-at_install")
```

Comment:
If it's possible you should avoid this. Post-install tests usually break in integration environments:

```suggestion
```